Fix broken build after upgrading listen

We have recently upgrade our listen gem here: https://github.com/tactilenews/100eyes/pull/605
For some reason, since then our build is failing with:

```rb
FATAL: Listen error: unable to monitor directories for changes.
Visit https://github.com/guard/listen/blob/master/README.md#increasing-the-amount-of-inotify-watchers for info on how to fix this.
Error: Process completed with exit code 1
```

In  rlease 3.3.4 though, listen has updated their README to address this issue. Apparently, it's not a new issue as they used to have it linked in their wiki, but it got deleted. It's not exactly clear why this bubbled up for the first time now, but @roschaefer has provided some code that addresses the issue for us here: https://github.com/tactilenews/100eyes/pull/600/files

Since it's not logically related to that dependency upgrade, this issue exists.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -27,6 +27,9 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Update file watchers because of rails settings cached
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+
       - name: Install ruby
         uses: actions/setup-ruby@v1
         with: { ruby-version: 2.7.2 }

